python/shaped_bloom_filter/golib/main.go

```diff
@@ -0,0 +1,178 @@
+package main
+
+/*
+#include <stdlib.h>
+#include <string.h>
+
+struct BloomFilter {
+	unsigned int m;
+    unsigned int k;
+	unsigned int b_length;
+	char *b;
+};
+*/
```

Comment:
remove?

Reply:
Cannot be removed because it's the preamble to the "C" library that we importing in this very file.